src/PolykeyClient.ts

```diff
@@ -13,6 +14,21 @@ import * as events from './events';
 import config from './config';
 import * as networkUtils from './network/utils';
 import clientClientManifest from './client/callers';
+import {option} from "fast-check";
```

Comment:
Importing `fast-check`? in a `src` file?

```suggestion
```

Reply:
lmao, ide auto import, not lintfixed

Reply:
Do not leave this in the codebase. It does not matter what IDE you use, you must maintain correctness as much as possible on every commit you're submitting, otherwise it wastes time for people reviewing.